### PR TITLE
Removing -recursive flag

### PR DESCRIPTION
**Pull request by fhnaseer**
-recrusive flag is removed. Inseated user can provide globbing patterns to find files. e.g. **\\*.coverage

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/additional-tools/dotnet-coverage.md](https://github.com/dotnet/docs/blob/a7807ede305437cd57ee5c4bfe74827827757399/docs/core/additional-tools/dotnet-coverage.md) | [dotnet-coverage code coverage utility](https://review.learn.microsoft.com/en-us/dotnet/core/additional-tools/dotnet-coverage?branch=pr-en-us-38525) |

### CODE REVIEW

**Review by gewarren — APPROVED** [review]
Do we need to keep the flag but mark it as only applicable to .NET 7 and earlier?

cc @tdykstra

**Comment by fhnaseer** [comment]
> > Do we need to keep the flag but mark it as only applicable to .NET 7 and earlier?
> 
> That would be preferable. But I would keep it only in the Options section, not in synopsis or examples.

Sure. Can you please show me an example (mentioning .NET 7 or earlier)?

File: docs/core/additional-tools/dotnet-coverage.md [hunk 1]
```diff
@@ -80,10 +80,6 @@ dotnet-coverage merge
 
   Removes all input coverage reports that were merged.
 
-* **`-r, --recursive`**
-
-  Search for coverage reports in subdirectories.
```
Comment (gewarren) [inline docs/core/additional-tools/dotnet-coverage.md:85]: I would keep lines 83-86, and change line 85 to this:

```
  [.NET 7 SDK and earlier versions only] Search for coverage reports in subdirectories.
```

Comment (fhnaseer) [reply docs/core/additional-tools/dotnet-coverage.md:85]: Updated.

**Pull request merged**